docs/installation.rst

```diff
@@ -23,10 +23,46 @@ Redis is the recommended storage type for Flask-Session, as it has the most comp
   Flask-Session versions below 1.0.0 (not yet released), use pickle_ as the default serializer, which may have security implications in production if your storage is ever compromised.
 
 
-Direct support
----------------
+Enhanced Installation Options
+-----------------------------
 
-Flask-Session has an increasing number of directly supported storage and client libraries.
+Flask-Session now supports a variety of storage backends directly through optional dependencies. This simplifies the installation process, allowing you to specify the required storage backend at the time of Flask-Session installation itself.
+
+Below is a guide on how to install Flask-Session with support for the desired storage backend:
+
```

Comment:
I think lines 26 to 32 are probably not needed as you said enough on line 13. Removing this would make it more concise.